RFS/src/main/java/com/rfs/cms/OpenSearchWorkCoordinator.java

```diff
@@ -123,7 +123,7 @@ public String toString() {
                             return "[ statusCode: " + r.getStatusCode() + ", payload: " + payloadStr + "]";
                         }
                     },
-                    (response, ignored) -> (response.getStatusCode() / 100) != 2);
```

Comment:
Just curious - what was the original logic you were trying to achieve here?  I didn't quite follow your explanation in the PR overview.

Reply:
The test determines when we're done needing to retry.  When this is 'true', the work is deemed to be done.  In other words, we would keep looping until we got something other than a 200.  Notice that the beginning of the method was a HEAD that didn't have the same affliction - so in the worst case scenario, we'd burn a process and be resolved the next time that the program ran